client/my-sites/domains/domain-management/domain-overview-pane/style.scss

```diff
@@ -148,3 +148,42 @@
 		padding: 0;
 	}
 }
+
+.domains-overview {
+	.hosting-dashboard-layout-with-columns__container {
+		background: var(--color-sidebar-background);
+	}
+
+	.hosting-dashboard-item-view {
+		background: var(--color-light-backdrop);
+		border-radius: 8px; /* stylelint-disable-line scales/radii */
+		flex-grow: 1;
+		max-height: calc(100vh - 32px - var(--masterbar-height));
+		transition: flex-grow 0.2s;
+		width: auto;
+	}
+
+	.hosting-dashboard-item-view__header-info {
+		flex-wrap: wrap;
+	}
+
+	.hosting-dashboard-item-view__content div {
+		flex-grow: 1;
```

Comment:
Hi @Imran92! This prop is quite generic, and it's affecting all my divs in the domain side of things. Can you narrow it to your needs please?

Reply:
Hey @alshakero 👋 It should be alright now, we've removed that style